src/main/java/de/zalando/aruha/nakadi/service/subscription/model/Session.java

```diff
@@ -0,0 +1,42 @@
+package de.zalando.aruha.nakadi.service.subscription.model;
+
+import java.util.UUID;
+
+public class Session {
+    public final String id;
```

Comment:
The reason is to follow Encapsulation principle, not to expose the inner structure of your class. 

>> I don't see any reasons to create getters that will just return field value.
In 99% cases getter just returns field value.